flow-nodes/api-builder-plugin-typeof/src/flow-nodes.yml

```diff
@@ -5,28 +5,38 @@ flow-nodes:
     description: Gets the type of data
     category: general
     methods:
-      getType:
+      getType: # This same key is used in `actions.js`
         name: Get Type
         description: Gets the type of data
         parameters:
           data:
+            name: Data
             description: The data that you want to find the type of.
             required: true
-        outputs:
-          # success output(s) should be first
-          next:
-            name: Next
-            context: $.type
             schema:
               type: string
-              enum:
-                - array
-                - bigint
-                - boolean
-                - function
-                - "null"
-                - number
-                - object
-                - string
-                - symbol
-                - undefined
+        # "Next" (next) output
+        returns:
+          name: Next
+          description: The operation was successful.
+          context: $.type
+          schema:
+            type: string
+            enum:
+              - array
+              - bigint
+              - boolean
+              - function
+              - "null"
+              - number
+              - object
+              - string
+              - symbol
+              - undefined
+        # "Error" (error) output
+        throws:
+          name: Error
+          description: An unexpected error was encountered.
+          context: $.error
+          schema:
+            type: string      
```

Comment:
In this case, any errors that are thrown aren't going to be strings, they'll be objects with a message

Reply:
Eh, I got news for ya - that's what is currently in the template.  Nice catch, but we have to fix this in all the flow-nodes :/

Reply:
Fixed this.